Arduino/ACAN2515.h

```diff
@@ -42,11 +42,13 @@ struct ACAN2515
   void receive(CANMessage & message);
   bool tryToSend(const CANMessage & message);
   void end();
+  uint16_t receiveBufferCount();
```

Comment:
This file is supporting unit tests.

Reply:
This file doesn't look anything like the ACAN2515.h that I use.

Reply:
No, this is just a dummy to make unit tests compile. I have only added the necessary declarations here. It does not come from the actual ACAN2515 library (which does not compile outside the Arduino environment).

Reply:
OK